src/hotspot/share/utilities/globalDefinitions.cpp

```diff
@@ -236,6 +236,17 @@ const char* type2name_tab[T_CONFLICT+1] = {
   "*narrowklass*",
   "*conflict*"
 };
+const char* type2name(BasicType t) {
+  if (t < (T_CONFLICT + 1)) {
```

Comment:
Looks cleaner to do `if (t <= T_CONFLICT)`, maybe?

Reply:
Hm, yeah. This looks like this to create symmetry with the declaration of `type2name_tab`:

```
extern const char* type2name_tab[T_CONFLICT+1];     // Map a BasicType to a char*
```

But the symmetry is lost here as they're in 2 different files now. I'd like a RFE where `type2name_tab` is no longer external which will make the symmetry apparent again. Can we ignore this nit in anticipation of that change?

Reply:
I honestly don't care about this nit either way :) You decide.

Reply:
Alright, I'll let it be then :).